feeds/exo.quebec.dmfr.json

```diff
@@ -131,75 +154,66 @@
         "url": "https://www.donneesquebec.ca/fr/licence/#cc-by",
         "use_without_attribution": "no",
         "create_derived_product": "yes"
-      },
-      "operators": [
-        {
-          "onestop_id": "o-f259-exo~citduhaut~saint~laurent",
-          "name": "EXO - Haut-Saint-Laurent",
-          "website": "http://www.cithsl.amt.qc.ca/index.asp",
-          "associated_feeds": [
-            {
-              "gtfs_agency_id": "CITHSL"
-            }
-          ]
-        }
-      ]
+      }
     },
     {
-      "id": "f-f25d-exo~citlerichelain",
-      "spec": "gtfs",
+      "id": "f-f259-exo~citduhaut~saint~laurent~rt",
+      "spec": "gtfs-rt",
       "urls": {
-        "static_current": "https://exo.quebec/xdata/citlr/google_transit.zip"
+        "realtime_vehicle_positions": "https://opendata.exo.quebec/ServiceGTFSR/VehiclePosition.pb?agency=cithsl",
+        "realtime_trip_updates": "https://opendata.exo.quebec/ServiceGTFSR/TripUpdate.pb?agency=cithsl",
+        "realtime_alerts": "https://opendata.exo.quebec/ServiceGTFSR/Alert.pb?agency=cithsl"
       },
       "license": {
         "spdx_identifier": "CC-BY-4.0",
-        "url": "https://www.donneesquebec.ca/fr/licence/#cc-by",
-        "use_without_attribution": "no",
-        "create_derived_product": "yes"
+        "url": "https://exo.quebec/en/about/open-data"
       },
-      "operators": [
-        {
-          "onestop_id": "o-f25d-exo~citlerichelain",
-          "name": "EXO - Le Richelain",
-          "website": "http://www.citrichelain.com/",
-          "associated_feeds": [
-            {
-              "gtfs_agency_id": "CITLR"
-            }
-          ]
-        }
-      ]
+      "authorization": {
+        "type": "query_param",
+        "param_name": "token",
+        "info_url": "https://exo.quebec/en/about/open-data/RequestAccessForm"
+      }
+    },
+    {
+      "id": "f-f25d-exo~reseaudetransportmetropolitain",
+      "spec": "gtfs",
+      "urls": {
+        "static_current": "https://exo.quebec/xdata/trains/google_transit.zip"
+      }
     },
     {
-      "id": "f-f25d-exo~citroussillon",
+      "id": "f-f25d-exo~richelain~roussillon",
+      "supersedes_ids": [
+        "f-f25d-exo~citroussillon",
+        "f-f25d-exo~reseaudetransportmetropolitain"
```

Comment:
@drewda I think the second supersedes_ids should be `f-f25d-exo~citlerichelain`? `reseaudetransportmetropolitain` is for commuter trains.

Reply:
@FelixINX thanks for catching this